### PR TITLE
Replace ‘scripts’ by Pharo script

### PR DESCRIPTION
**Pull request by Rinzwind**
This pull request replaces the scripts in the ‘scripts’ directory by a Pharo script in the ‘README.md’ file. By default it only generates the PNG files that don’t already exist, so that they are only generated for newly added icons. The variable ‘shouldRegenerateAll’ can be set to ‘true’ to regenerate them for all icons. Inkscape seems to treat a missing source SVG file as a warning, rather than error, condition and doesn’t exit with a non-zero status in that case, hence the script explicitly checks that the PNG file exists after executing the shell command for generating it.

### CODE REVIEW

**Comment by Rinzwind** [comment]
@jecisc Could you maybe check whether this can be merged?

File: README.md [hunk 1]
```diff
@@ -4,4 +4,349 @@ Based on Idea11 icons from [pharo-project](https://github.com/pharo-project/phar
 Created using Inkscape.
 
 The folder ```svg``` contains icons in format svg.
-PNG files are generated from these svg files executing the script ```scripts/EXPORT-TO-PNG.sh```
+PNG files are generated from these svg files by executing this script:
+
+```smalltalk
```
Comment (astares) [inline README.md:9]: This is bound to MacOS and not usable on other platforms (like Windows)

Comment (Rinzwind) [reply README.md:9]: I’m a bit confused because that doesn’t seem to be a change from the scripts as previously written?

Comment (Rinzwind) [reply README.md:9]: @astares Can I remind you about the above?

Comment (astares) [reply README.md:9]: @Rinzwind OK - fine for me.

**Review by astares — APPROVED** [review]

**Comment by Ducasse** [comment]
This is nice. 
Could we remove the 'glamourous' strings from the name since we do not want to have reference to glamourous in Pharo?

**Comment by Rinzwind** [comment]
I’m not sure that’s something to be dealt with in this pull request, I opened an issue about it: [Pharo issue #17017](https://github.com/pharo-project/pharo/issues/17017).

**Comment by Rinzwind** [comment]
I’m not sure what the status here now is, though I assume it can be merged?

**Comment by Rinzwind** [comment]
@jecisc @astares @Ducasse: Could one of you maybe check whether this can now be merged?

**Comment by Rinzwind** [comment]
Thanks!

**Pull request merged**